operator/watchers/cilium_endpoint.go

```diff
@@ -102,7 +146,10 @@ func convertToCiliumEndpoint(obj interface{}) interface{} {
 				OwnerReferences: concreteObj.OwnerReferences,
 			},
 			Status: cilium_api_v2.EndpointStatus{
-				Identity: concreteObj.Status.Identity,
+				Identity:   concreteObj.Status.Identity,
+				Networking: concreteObj.Status.Networking,
+				NamedPorts: concreteObj.Status.NamedPorts,
+				Encryption: concreteObj.Status.Encryption,
```

Comment:
Similar changes are needed on line 126

Reply:
Can bit more descriptive ? Not sure what is missing and where we need.

Reply:
Check a couple lines bellow, the `dfsu := cache.DeletedFinalStateUnknow` struct misses these fields as well.